File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;

class Solution {
    public String solution(String s) {
        StringTokenizer st = new StringTokenizer(s);
        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;
        while (st.hasMoreTokens()) {
            int k = Integer.parseInt(st.nextToken());
            max = Math.max(max, k);
            min = Math.min(min, k);
        }
        return min + " " + max;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <vector>
#include <sstream>
#include <climits> // INT_MAX, INT_MIN

using namespace std;

string solution(string s) {
    stringstream ss(s);
    
    int num;
    int minValue = INT_MAX;
    int maxValue = INT_MIN;
    
    while (ss >> num) {
        if (num < minValue) {
            minValue = num;
        }
        if (num > maxValue) {
            maxValue = num;
        }
    }
    
    return to_string(minValue) + " " + to_string(maxValue);
}